src/App.js

```diff
@@ -1,7 +1,33 @@
-import "./App.css";
+import { Routes, Route } from 'react-router';
```

Comment:
- [ ] I couldn't find your video presentation, please be reminded that it is mandatory and must fulfill the following requirements:
![image](https://github.com/rahimialijan/React-Capstone-Project/assets/62705947/d20648d2-a683-4406-8fb2-ca6b454685f9)

Reply:
done